tools/ci_testing/addons_cpu.sh

```diff
@@ -44,7 +44,7 @@ export TF_NEED_CUDA=0
 export PYTHON_BIN_PATH=`which python`
 ls -alh $PYTHON_BIN_PATH
 # Use default configuration here.
-yes 'y' | ./configure.sh
+echo 'y' | ./configure.sh
```

Comment:
A little curious: why use echo instead here?

Reply:
The windows shell was hanging when using 'yes' to step through configure.sh. Because linux/mac/windows use `addons_cpu.sh` this was switched.

https://github.com/tensorflow/addons/blob/master/tools/ci_build/builds/release_windows.sh#L31

Reply:
It's a good call-out though, having to pass a single echo is a bit brittle since it only works when `TF_NEED_CUDA` is set. Open to changes for sure